backends-common/pulsar/src/test/java/org/apache/james/backends/pulsar/PulsarConfigurationTest.java

```diff
@@ -155,8 +158,106 @@ void fromShouldReturnTheConfigurationWhenRequiredParametersAreGiven() {
 
         configuration.addProperty("namespace", namespace);
         assertThat(PulsarConfiguration.from(configuration))
-                .isEqualTo(new PulsarConfiguration(brokerUri, adminUri, new Namespace(namespace)));
+                .isEqualTo(new PulsarConfiguration(brokerUri, adminUri, new Namespace(namespace), Auth.noAuth()));
     }
+    @Test
+    void fromShouldThrowWithTokenAuthenticationWhenTokenIsMissing() {
```

Comment:
I don't see the test that it returns successfully the conf if token auth params are valid

Reply:
fixedup